.gitignore

```diff
@@ -21,3 +21,32 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+build
+.gradle/
+target
+
+# IDEA Files
+.idea/
+*.iml
+*.ipr
+*.iws
+
+# VSCode files
+.project
+.classpath
+.settings
+bin/
+
+# MacOS
+*.DS_Store
+
+# Ballerina
+velocity.log*
+*Ballerina.lock
+
+# Auto generated Dependencies.toml files
+examples/**/Dependencies.toml
+load-tests/**/Dependencies.toml
+integration-tests/Dependencies.toml
+ballerina-tests/Dependencies.toml
```

Comment:
Let's remove these if there aren't such directories.

Reply:
Updated in [ba096c3](https://github.com/ballerina-platform/persist-tools/pull/9/commits/ba096c3982f7ac14ba6d37daf954b2f14e50f23c)